exifinterface/exifinterface/src/main/java/androidx/exifinterface/media/ExifInterface.java

```diff
@@ -6774,8 +6774,8 @@ private void saveWebpAttributes(InputStream inputStream, OutputStream outputStre
 
                         // Retrieve image width/height
                         widthAndHeight = totalInputStream.readInt();
-                        width = (widthAndHeight << 18) >> 18;
-                        height = (widthAndHeight << 2) >> 18;
+                        width = widthAndHeight & 0x3FFF;
+                        height = (widthAndHeight >> 16) & 0x3FFF
```

Comment:
Please can you add the missing semi-colon at the end of this line

Reply:
Sorry for my mistake. I love Kotlin lol